### ISSUE TITLE
Multiple server fail

### ISSUE THREAD

**Issue by superstorm192** [manicdigger/manicdigger#214]
I have copied my manic digger folder and changed the copy's port. But when I run both servers at the same time, we get an identity crisis. Only one of the servers show up in the server list on both the website and the in game list. And sometimes you get sent to the OTHER server. Link saying Server1 but instead goes to Server2.

Ports 25565 and 25564

**Comment by croxxx** [comment]
Please note that for running 2 servers you will need two open ports (which I guess you have) and **different GUIDs for each server**.
Simply delete (or rename) the `ServerConfig.txt` for a new one to be generated. Do not edit the GUID manually!

**Comment by superstorm192** [comment]
so is it `<key>` that needs generated? could i simply copy the new `<key>` over to the original `ServerConfig`? Imma test that out and see what happens.

**Comment by superstorm192** [comment]
Tested it now, different ports, different file folders, different `<key>`s and there seems to no longer be an identity crisis. Thank you so much :D!

Here is what I did, and was a success!
I opened the unchanged `ServerConfig.txt` from on Server#2 in wordpad
I then deleted `ServerConfig.txt` and ran `ManicDiggerServer.exe` (Server#2)
I opened the new default `ServerConfig.txt` and copied the `<key>`
I pasted it into the unchanged `ServerConfig.txt` that I opened before I deleted it and saved it
Now `ServerConfig.txt` is exactly the same as before except it has a new key generated by `ManicDiggerServer.exe`
